dialect/sql/schema/migrate.go

```diff
@@ -59,14 +59,23 @@ func WithFixture(b bool) MigrateOption {
 	}
 }
 
+// WithForeighKeysDisable disables creating foreigh-key in ddl. In some case we are
+// not allowed to use foreigh-key. Defaults to false
+func WithForeighKeysDisable(b bool) MigrateOption {
+	return func(m *Migrate) {
+		m.disableForeighKeys = b
+	}
+}
```

Comment:
What do you think about renaming it to `WithForeignKeys` and use `true` as default?

The usage will be as follows:
```go
    err := client.Schema.Create(
        ctx, 
        // Disable FKs creation.
        migrate.WithForeignKeys(false),
    )
```